WordPress/src/main/java/org/wordpress/android/ui/posts/PostsListFragment.java

```diff
@@ -696,4 +705,15 @@ public void onMediaChanged(MediaStore.OnMediaChanged event) {
             }
         }
     }
+
+    @SuppressWarnings("unused")
+    @Subscribe(threadMode = ThreadMode.MAIN)
+    public void onMediaUploaded(OnMediaUploaded event) {
+        if (event.media != null && event.completed) {
+            PostModel post = mPostStore.getPostByLocalPostId(event.media.getLocalPostId());
+            if (post != null && PostUploadService.isPostUploadingOrQueued(post)) {
+                loadPosts(LoadMode.FORCED);
```

Comment:
Disregard this comment - performed some tests to realize how it's actually working 👍